Source/Particles/MultiParticleContainer.cpp

```diff
@@ -236,15 +236,14 @@ MultiParticleContainer::ReadParameters ()
             }
 
         }
-        pp.query("use_fdtd_nci_corr", WarpX::use_fdtd_nci_corr);
+        pp_particles.query("use_fdtd_nci_corr", WarpX::use_fdtd_nci_corr);
 #ifdef WARPX_DIM_RZ
         AMREX_ALWAYS_ASSERT_WITH_MESSAGE(WarpX::use_fdtd_nci_corr==0,
                             "ERROR: use_fdtd_nci_corr is not supported in RZ");
 #endif
-        pp.query("galerkin_interpolation", WarpX::galerkin_interpolation);
```

Comment:
Note that this wrong query is already removed in #1808. Until #1808 is merged, I have to remove it here too, in order to avoid build errors.